helpers/atelierAPI.js

```diff
@@ -0,0 +1,41 @@
+const axios = require('axios');
+const config = require('../config.js');
```

Comment:
for this to work with everyones file structures youll need to write this line using {token} again

Reply:
Changed to `{ token, url }`.